src/main/scala/edu/ie3/simona/model/participant/SystemParticipant.scala

```diff
@@ -101,7 +101,10 @@ abstract class SystemParticipant[
       val activePower = calculateActivePower(data)
       val reactivePower =
         calculateReactivePower(activePower, voltage)
-      ApparentPower(activePower, reactivePower)
+      ApparentPower(
+        activePower * scalingFactor,
+        reactivePower * scalingFactor
+      )
```

Comment:
We decided to apply the scaling factor to all outputs of the participant, including e.g. heat. So currently this would have to be implemented in `ApparentPowerAndHeatParticipant` I guess